data/members/classy.json

```
@@ -0,0 +1,5 @@
+{
+  "name": "classy"
+  "github": "C1455Y"
```

```suggestion
  "name": "classy",
  "github": "C1455Y",
```
make sure lines are separated with commas